diff --git a/share/tables/index.php b/share/tables/index.php
index 693f35a..fdffb39 100644
--- a/share/tables/index.php
+++ b/share/tables/index.php
@@ -1,4 +1,7 @@
 <html>
+<head>
+<link rel="icon" href="data:;base64,iVBORw0KGgo=">
+</head>
 <body>
 <form action=>
 <select name="nodename" size="1">
@@ -32,13 +35,16 @@ function nodename( $filename ) {
 
 $dirname = './';
 $dir = opendir($dirname);
+$array_files = array();
 while($file = readdir($dir)) {
     if($file != '.' && $file != 'index.php' && $file != '.index.php.swp' && $file != '..' && !is_dir($dirname.$file)) {
         #echo '<option value='.$file.'>'.nodename($file).'</option>';
         $array_files[$file] = nodename($file);
     }
 }
-asort($array_files);
+if (!empty($array_files)) {
+    asort($array_files);
+}
 foreach ($array_files as $key => $val) {
     echo '<option value='.$key.'>'.$val.'</option>';
 }
@@ -51,7 +57,7 @@ closedir($dir);
 </form><br>
 
 <?php
-if (isset($_GET['nodename'])) {
+if (isset($_GET['nodename']) and !empty($_GET['nodename'])) {
     $nodename=$_GET['nodename'];
     include("$nodename");
 }
