diff --git a/web/graph.d/network_report.php b/web/graph.d/network_report.php
index 507831b..09b151e 100644
--- a/web/graph.d/network_report.php
+++ b/web/graph.d/network_report.php
@@ -25,11 +25,13 @@ function graph_network_report ( &$rrdtool_graph ) {
        $rrdtool_graph['title'] = "$hostname Network last $range";
     }
     $rrdtool_graph['lower-limit']    = '0';
-    $rrdtool_graph['vertical-label'] = 'Bytes/sec';
+    $rrdtool_graph['vertical-label'] = 'Bits/sec';
     $rrdtool_graph['extras']         = '--rigid --base 1024';
 
-    $series = "DEF:'bytes_in'='${rrd_dir}/bytes_in.rrd':'sum':AVERAGE "
-       ."DEF:'bytes_out'='${rrd_dir}/bytes_out.rrd':'sum':AVERAGE "
+    $series = "DEF:'bytes_in_def'='${rrd_dir}/bytes_in.rrd':'sum':AVERAGE "
+       ."DEF:'bytes_out_def'='${rrd_dir}/bytes_out.rrd':'sum':AVERAGE "
+       ."CDEF:'bytes_in'='bytes_in_def,8,*' "
+       ."CDEF:'bytes_out'='bytes_out_def,8,*' "
        ."LINE2:'bytes_in'#$mem_cached_color:'In' "
        ."LINE2:'bytes_out'#$mem_used_color:'Out' ";
 
diff --git a/web/templates/default/meta_view.tpl b/web/templates/default/meta_view.tpl
index eaec410..1a3ac9a 100644
--- a/web/templates/default/meta_view.tpl
+++ b/web/templates/default/meta_view.tpl
@@ -2,7 +2,7 @@
 
 <!-- START BLOCK : source_info -->
 <TR>
-  <TD CLASS={class} COLSPAN=3>
+  <TD CLASS={class} COLSPAN=4>
    <A HREF="{url}"><STRONG>{name}</STRONG></A> {alt_view}
   </TD>
 </TR>
@@ -27,12 +27,19 @@
   </A>
   </TD>
 
-  <TD VALIGN=top>
+  <TD VALIGN=top align=right>
   <A HREF="{url}" VALIGN=top>
    <IMG SRC="./graph.php?{graph_url}&g=mem_report&z=medium&r={range}"
        ALT="{name} MEM" BORDER="0">
   </A>
    </TD>
+
+  <TD VALIGN=top align=right>
+  <A HREF="{url}" VALIGN=top>
+   <IMG SRC="./graph.php?{graph_url}&g=network_report&z=medium&r={range}"
+       ALT="{name} NET" BORDER="0">
+  </A>
+   </TD>
 <!-- END BLOCK : public -->
 
 <!-- START BLOCK : private -->
